data_model_Calix-EC4F82-ENT.json

```diff
@@ -1,1035 +1,4805 @@
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap." : [false],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.1." : [false],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.1.DigitMap" : [true, "[ABCD*#X].T", "xsd:string"],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.1.Type" : [false, "0", "xsd:unsignedInt"],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.1.Route" : [true, "0", "xsd:unsignedInt"],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.2." : [false],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.2.DigitMap" : [true, "2222", "xsd:string"],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.2.Type" : [false, "1", "xsd:unsignedInt"],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.2.Route" : [true, "1", "xsd:unsignedInt"],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.3." : [false],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.3.DigitMap" : [true, "3333", "xsd:string"],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.3.Type" : [false, "2", "xsd:unsignedInt"],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.3.Route" : [true, "0", "xsd:unsignedInt"],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.4." : [false],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.4.DigitMap" : [true, "963", "xsd:string"],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.4.Type" : [false, "3", "xsd:unsignedInt"],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.NumberingPlan.X_HUAWEI_DigitMap.4.Route" : [true, "2", "xsd:unsignedInt"],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.ButtonMap." : [false],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.ButtonMap.Button." : [true],
-"InternetGatewayDevice.Services.VoiceService.1.VoiceProfile.1.Line." : [true],
-"InternetGatewayDevice.X_HUAWEI_FireWall." : [true],
-"InternetGatewayDevice.X_HUAWEI_FireWall.CurrentLevel" : [true, "Low", "xsd:string"],
-"InternetGatewayDevice.X_HUAWEI_FireWall.FilterMode." : [true],
-"InternetGatewayDevice.X_HUAWEI_FireWall.FilterMode.1." : [true],
-"InternetGatewayDevice.X_HUAWEI_FireWall.FilterMode.1.Level" : [true, "Disable", "xsd:string"],
-"InternetGatewayDevice.X_HUAWEI_FireWall.FilterMode.1
```

Comment:
A bunch of these values aren't correct
`[false,"","0"],` note: the `0`. end result should look something like instead of a `[false,"","xsd:string"]`